scripts/zones/Carpenters_Landing/mobs/Para.lua

```diff
@@ -0,0 +1,97 @@
+-----------------------------------
+--  Area: Carpenters' Landing
+--    NM: Para
+-- Quest: Elderly Pursuits
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180) -- 3 minutes
+
+    local para = GetMobByID(ID.mob.PARA)
+
+    if
+        Para:getLocalVar('spawnAdds') < 5
+    then
+        Para:setLocalVar('spawnAdds', Para:getLocalVar('spawnAdds') + 1)
+    end
+
+end
```

Comment:
Did you test this? Lua is case sensitive, so `para` is not the same as `Para`. Your log will have complained about using a nil object.

Typically, we don't have local vars with uppercase leading letters, reserving that for Globally bound C++ functions. Hopefully CI catches that